merlin/models/tf/inputs/continuous.py

```diff
@@ -27,12 +27,17 @@
     TabularBlock,
 )
 from merlin.models.utils.doc_utils import docstring_parameter
-from merlin.schema import Schema
+from merlin.schema import Schema, Tags
 
 
 @tf.keras.utils.register_keras_serializable(package="merlin.models")
 class Continuous(Filter):
-    ...
+    def __init__(
```

Comment:
can we add a one line explanation about this class?

Reply:
Sure. Added docstrings